CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.9.4 (09.08.2021)
+
+## Fixes
+
+#### ([1361](https://github.com/allegro/hermes/pull/1361)) Topic name is now published in tags by MicrometerTaggedMetricsProvider
```

Comment:
Considering that the changes we introduced there are not backward compatible, we should add info about them here. It would be nice to mention about the new metrics format and the new name of MicrometerMetricsProvider.